### PR TITLE
SLING-11408 m-assembly-p must use dependencies generated from

### PR DESCRIPTION
**Pull request by kwin**
appassembler

### CODE REVIEW

**Comment by cziegeler** [comment]
Thanks, this makes the launcher now start, but all logs from the launcher are now gone, so I guess there is something wrong with slf4j dependencies

**Comment by cziegeler** [comment]
ok, so additional problems:

1. slf4j-impl is missing
2. the launch script now lists all jar files from libs as the classpath, whereas previously a wildcard was used, which allowed to just add new jars in the libs directory

**Comment by kwin** [comment]
Probably https://github.com/apache/sling-org-apache-sling-feature-launcher/blob/072fe2ae0fa1ae0a931c546aa25ec60a4fc75194/pom.xml#L259 needs to have compile or runtime scope. Appassembler by default only considers Maven runtime scope.

**Comment by cziegeler** [comment]
I fixed the two issues

**Review by cziegeler — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -256,7 +257,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <scope>provided</scope>
+            <scope>compile</scope>
```
Comment (kwin) [inline pom.xml:260]: I would rather have this scope runtime, as definitely not needed in the compile class path.

Comment (cziegeler) [reply pom.xml:260]: Feel free to adjust if it is still working then

Comment (kwin) [reply pom.xml:260]: Done in https://github.com/apache/sling-org-apache-sling-feature-launcher/commit/b55ca9cb6d3019aef854c45e717eb476e874c4d0.

**Pull request merged**